Bloon/Core/Discord/Bot.cs

```diff
@@ -71,6 +80,8 @@ public override async Task Enable()
             this.cNext.RegisterCommands<GeneralCommands>();
             this.cNext.RegisterCommands<OwnerCommands>();
 
+            this.slash.RegisterCommands<LTPSlashCommand>(Guilds.SBG);
```

Comment:
Move to feature

Reply:
can't doesn't work for whatever purpose.